backend/src/main/java/com/bakdata/conquery/models/config/SqlConnectorConfig.java

```diff
@@ -26,6 +27,7 @@ public class SqlConnectorConfig {
 	/**
 	 * Keys must match the name of existing {@link Dataset}s.
 	 */
+	@NonNull
 	@Getter(AccessLevel.PRIVATE)
 	private Map<String, DatabaseConfig> databaseConfigs;
```

Comment:
Ich glaube hier muss noch eine annotation hin, sonst schaut der Validator nicht in die Map
```suggestion
	@NonNull
	@Getter(AccessLevel.PRIVATE)
	private Map<String, @Valid DatabaseConfig> databaseConfigs;
```